complex/ft_test.c

```diff
@@ -262,10 +262,55 @@ static int ft_sync_test(int value)
 	return ft_sock_sync(value);
 }
 
+static int ft_pingpong_rma(void)
+{
+	int ret, i;
+	size_t msg_size = ft_tx_ctrl.msg_size;
+
+	if (listen_sock < 0) {
+		for (i = 0; i < ft_ctrl.xfer_iter; i++) {
+			ft_tx_ctrl.msg_size = msg_size;
+			ret = ft_send_rma();
+			if (ret)
+				return ret;
+
+			ft_tx_ctrl.msg_size = ft_ctrl.size_array[0];
+			ret = ft_send_msg();
+			if (ret)
+				return ret;
+
+			ret = ft_recv_msg();
+			if (ret)
+				return ret;
+		}	
+	} else {
+		for (i = 0; i< ft_ctrl.xfer_iter; i++) {
+			ft_tx_ctrl.msg_size = msg_size;
```

Comment:
I don't think we need to set this until sending the rma.  I don't like the weirdness of changing the msg_size like this, but the alternatives are to either duplicate ft_send/recv_msg or change the calls to pass the size through.